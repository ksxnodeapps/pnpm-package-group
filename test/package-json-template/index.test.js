'use strict'
const data = require('../../lib/package-json-template/data')
const main = require('../../lib/package-json-template')

describe('data', () => {
  it('stays unchanged', () => {
    expect(data).toMatchSnapshot()
  })
})

describe('main module', () => {
  it('stays unchanged', () => {
    expect(main).toMatchSnapshot()
  })

  it('has .object', () => {
    expect(main.object).toEqual(data)
  })

  it('has .json', () => {
    expect(JSON.parse(main.json)).toEqual(data)
  })
})
