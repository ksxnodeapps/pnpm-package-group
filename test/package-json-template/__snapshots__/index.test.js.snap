// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data stays unchanged 1`] = `
Object {
  "description": "Just a transational directory",
  "name": "TransationalDirectory",
  "private": true,
  "version": "0.0.0",
}
`;

exports[`main module stays unchanged 1`] = `
Object {
  "json": "{
  \\"name\\": \\"TransationalDirectory\\",
  \\"version\\": \\"0.0.0\\",
  \\"description\\": \\"Just a transational directory\\",
  \\"private\\": true
}",
  "object": Object {
    "description": "Just a transational directory",
    "name": "TransationalDirectory",
    "private": true,
    "version": "0.0.0",
  },
}
`;
